controller/instance_manager_controller.go

```diff
@@ -807,6 +807,11 @@ func (imc *InstanceManagerController) createInstanceManagerPod(im *longhorn.Inst
 		return err
 	}
 
+	podSpec.Spec.Containers[0].Env = append(podSpec.Spec.Containers[0].Env,
+		v1.EnvVar{Name: "GRPC_GO_LOG_VERBOSITY_LEVEL", Value: "99"},
```

Comment:
I feel this is quite aggressive. As for debugging, but what the major reason we should turn on the verbosity and use debug level instead of info?

Reply:
This is marked as DEV, I left the commit in for the review but indeed to remove it before merging.
The dev commits make it easier to review/test this out as well to get a better understanding on how the connection setup/teardown reconnection etc works.